### PR TITLE
doc improvement: Add using Programmer to nRF9160 DK section

### PR DESCRIPTION
**Pull request by wiba-nordic**
The section for Building ad Programming an application in
the Developing with (was Working with) nRF9160 DK section
was very brief.

This PR adds additional information for using the Programmer
app to program application firmware to that section.

Signed-off-by: Wille Backman <wille.backman@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_nrf9160.rst [hunk 1]
```diff
@@ -185,7 +185,16 @@ Make sure to select a suitable build target when building your application.
 Building and programming
 ************************
 
-See :ref:`gs_programming` for information on building and programming samples on nRF9160 DK.
+You can program applications and samples on the nRF9160 DK after obtaining the corresponding firmware images.
+
+Download the latest application and modem firmware from `nRF9160 DK Downloads <nRF9160 product website (compatible downloads)>`_ page.
+When you update the application firmware on an nRF9160 DK, you must also update the modem firmware as described in :ref:`nrf9160_gs_updating_fw_modem`.
+
+To program applications using the Programer app from `nRF Connect for Desktop`_, follow the instructions in :ref:`nrf9160_gs_updating_fw_application`.
```
Comment (umapraseeda) [inline doc/nrf/ug_nrf9160.rst:193]:
```suggestion
To program applications using the Programmer app from `nRF Connect for Desktop`_, follow the instructions in :ref:`nrf9160_gs_updating_fw_application`.
```

File: doc/nrf/ug_nrf9160.rst [hunk 2]
```diff
@@ -185,7 +185,16 @@ Make sure to select a suitable build target when building your application.
 Building and programming
 ************************
 
-See :ref:`gs_programming` for information on building and programming samples on nRF9160 DK.
+You can program applications and samples on the nRF9160 DK after obtaining the corresponding firmware images.
+
+Download the latest application and modem firmware from `nRF9160 DK Downloads <nRF9160 product website (compatible downloads)>`_ page.
+When you update the application firmware on an nRF9160 DK, you must also update the modem firmware as described in :ref:`nrf9160_gs_updating_fw_modem`.
```
Comment (umapraseeda) [inline doc/nrf/ug_nrf9160.rst:191]: Should we move this below line 193?

Comment (wiba-nordic) [reply doc/nrf/ug_nrf9160.rst:191]: It could also go as a Note on line 196.

File: doc/nrf/ug_nrf9160.rst [hunk 3]
```diff
@@ -185,7 +185,16 @@ Make sure to select a suitable build target when building your application.
...
+
+To program applications using the Programer app from `nRF Connect for Desktop`_, follow the instructions in :ref:`nrf9160_gs_updating_fw_application`.
+In Step 2, set the switch to **nRF91** or **nRF52** as appropriate for the application or sample you are programming.
+Likewise, in Step 7, choose the ``.hex`` file for the application you are programing.
```
Comment (umapraseeda) [inline doc/nrf/ug_nrf9160.rst:195]:
```suggestion
Likewise, in Step 7, choose the :file:`.hex` file for the application you are programming.
```

**Review by umapraseeda — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**